svg/pservers/reftests/hatch-overflow-001.svg

```diff
@@ -0,0 +1,52 @@
+<svg id="svg-root"
+  width="100%" height="100%" viewBox="0 0 480 360"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:html="http://www.w3.org/1999/xhtml">
+  <g id="testmeta">
+    <title>Hatch: Basic hatch.</title>
```

Comment:
This test seems to be testing that the default value of `overflow` on a hatch is `visible`. But the spec says that default overflow is hidden, consistent with patterns:

https://svgwg.org/svg-next/pservers.html#HatchNotes

More general comment: it would be useful if the tests had unique titles and/or assertions explaining what each one is testing.